harmony/client.py

```diff
@@ -222,7 +224,9 @@ def _params(self, request: BaseRequest) -> dict:
         skipped_params = ['shapefile']
         if isinstance(request, OgcBaseRequest):
             if request.is_edr_request():
-                params['forceAsync'] = True
+                # use string in lowercase as value to match how boolean values
+                # in params are converted below
+                params['forceAsync'] = 'true'
```

Comment:
I'd prefer if we used booleans throughout for all boolean parameters and treat it as an error if a string or any other type is used.

We can add the boolean validations to this function:
``` 
def error_messages(self) -> List[str]:
        """A list of error messages, if any, for the request."""
```

Reply:
This is used to submit to Harmony which has to be a string now.